apps/cli/src/app.ts

```diff
@@ -37,7 +38,12 @@ export interface App {
   consoleLogger: ConsoleLogger;
 }
 
-export function createApp(timeout?: Timeout): App {
+interface AppConfig {
+  timeout?: Timeout,
+  userWorkspace: Workspace
```

Comment:
Let's make this optional, so that way the `cli.ts` implementation doesn't need to change.